card.io/src/main/jni/Android.mk

```diff
@@ -11,16 +11,16 @@ LOCAL_PATH := $(call my-dir)
 LOCAL_DMZ_DIR := card.io-dmz
 
 # --- declare opencv prebuilt static libs ---------------------------------
-ifneq (,$(filter $(TARGET_ARCH_ABI),armeabi-v7a))
+ifneq (,$(filter $(TARGET_ARCH_ABI),armeabi-v7a x86))
```

Comment:
This syntax was already used in Android.mk:79: https://github.com/card-io/card.io-Android-source/blob/master/card.io/src/main/jni/Android.mk#L79